pkg/ext/apiserver_test.go

```diff
@@ -711,11 +709,11 @@ func setupExtensionAPIServer[
 	err = extensionAPIServer.Run(ctx)
```

Comment:
`setupExtensionAPIServer` both returns an error in some cases, **and** fails the test in other cases.

I suspect it doesn't need to return the error, it could just `require.NoError()`  and _maybe_ `t.Helper()` is appropriate in this case too?

Reply:
Sounds fair enough, we never really do anything with the errors from `setupExtensionAPISever` any way so we can just fail if we encounter one. And yes `t.Helper()` will be useful to us!